apps/src/p5lab/spritelab/blocks.js

```diff
@@ -377,7 +378,12 @@ export default {
           .appendTitle(fieldLabel, 'VAR')
           .appendTitle(Blockly.Msg.VARIABLES_GET_TAIL);
 
-        if (Blockly.useModalFunctionEditor) {
+        let allowBehaviorEditing = Blockly.useModalFunctionEditor;
+        if (appOptions.level.toolbox && !Blockly.hasCategories) {
```

Comment:
Can you add a comment explaining why we do this check? (No need to wait for drone after)